chatminer/chatparsers.py

```diff
@@ -240,6 +240,54 @@ def _parse_message(self, mess):
         return parsed_message
 
 
+class InstagramChatsParser(Parser):
+    def _read_file_into_list(self):
+        self._logger.info("Starting reading raw messages into memory...")
+        self.messages = []
+        with self._file.open(encoding="utf-8") as f:
+            messages_raw = reversed((json.load(f)["messages"]))
+
+        for line in messages_raw:
+            self.messages.append(line)
+        self._logger.info(
+            "Finished reading %i raw messages into memory.", len(self.messages)
+        )
+
+    def _parse_message(self, mess):
+        if "share" in mess:
+            body = "share"
+        elif "photos" in mess:
+            body = "photo"
+        elif "videos" in mess:
+            body = "video"
+        elif "audio_files" in mess:
+            body = "audio"
+        elif "content" in mess:
+            if any(
+                flag in mess["content"]
+                for flag in (
+                    " to your message",
+                    " in the poll.",
+                    " created a poll: ",
+                    " liked a message",
+                    "This poll is no longer available.",
+                    "'s poll has multiple updates.",
+                )
+            ):
+                return None
+            else:
+                body = mess["content"]
+        else:
+            return None
```

Comment:
```suggestion
        else:
            self._logger.warning("Skipped message with unknown format: %s", mess)
            return None
```

Reply:
I used to have this but there are a lot of messages that just don't have any content, so it spams the log. If you want me to add this though, I can.

Reply:
Are we able to catch the messages without any content in a previous step? 
This last else switch should only be the fallback for pattern we haven't seen in a log.

Reply:
I'll add a catch for no content.

Reply:
Just for clarification: For those empty messages, there is simply no field other than author and timestamp? 
If that's the case, what do you think of checking if we only have those to fields, and if, we put something like _"Empty message: Non-persistent media."_ into the message body?

Unfortunately, we can't be sure that we catch all the message types, as the platforms from time to time change the output format. So I would vote for putting the log message in to raise such cases.

Reply:
> Just for clarification: For those empty messages, there is simply no field other than author and timestamp?

Exactly. There is a "reactions" field as well, but only if there are reactions.

> If that's the case, what do you think of checking if we only have those to fields, and if, we put something like "Empty message: Non-persistent media." into the message body?

That might cause unwanted words in the wordcloud. I could make it say "photo" if you want though, even though it could be a video.

> the platforms from time to time change the output format

I didn't know that, but it makes sense. In that case I agree with having a log. I'll commit that right now.

Also, do you want me to change the output if there is a photo, share, video, or audio to something different? This might be a good idea because those words have a big bump in the wordcloud. Maybe something like "sent-photo" (with the hyphen so that it's one word)?

```python
if "share" in mess:
    body = "sent-share"
elif "photos" in mess:
    body = "sent-photo"
elif "videos" in mess:
    body = "sent-video"
elif "audio_files" in mess:
    body = "sent-audio"
```

Reply:
> That might cause unwanted words in the wordcloud.
This is why we have the `stopwords` parameter in the wordcloud, which we introduced since other parsers suffer from a similar issue. Thus, it should be fine to use the message as proposed.
I'll merge the PR as soon as you're happy with it 🙂

Reply:
Can I add some default stopwords? It doesn't look like there are any yet, but I don't see why adding 
```python
stopwords = STOPWORDS.update(["sent-share", "sent-photo", "sent-video", "sent-audio"])
```
after adding the user-inputted stopwords wouldn't work.

Reply:
Actually, there is a problem with the solution, as seen in #68.